lib/reference/selectors.cjs

```
@@ -10,21 +10,26 @@ const deprecatedHtmlTypeSelectors = new Set([
 	'applet',
 	'basefont',
 	'big',
+	'bgsound',
 	'blink',
 	'center',
 	'content',
 	'dir',
 	'font',
 	'frame',
 	'frameset',
-	'hgroup',
+	'image',
```

was silently converted <img>